packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -43,6 +43,7 @@ const StoryPromoWrapper = styled.div`
     display: grid;
     grid-template-columns: repeat(6, 1fr);
     grid-column-gap: ${GEL_GUTTER_BELOW_600PX};
+    grid-row-gap: ${GEL_GUTTER_BELOW_600PX};
```

Comment:
Does the top story styling only work for those with `grid` support??

Reply:
Spoke IRL, non-grid support needs to be added. Based on this we shouldn't use `grid-row-gap` and should have padding-bottom as it will work for both grid and non-grid